### DISCUSSION TITLE
Using Environment Variables

### DISCUSSION THREAD

**Discussion by evertonpavan** [ltb-project/self-service-password#526] [General]
Hello guys!
Is there some way to put the credentiais in some  .env file, then catch e load in config.inc.local.php file?

for example:

// .env file
LDAP_URL=ldap://216.485.774.221
ADMIN_AD=cn=Administrator,cn=Users,dc=sam,dc=dom
PASSWORD_AD=password

// config.inc.local.php
...
$ldap_url = LDAP_URL //from .env file
$ldap_binddn = ADMIN_AD //from .env file
$ldap_bindpw = PASSWORD_AD //from .env file
...

Regards,

Everton

**Comment by coudot** [comment]
You can use `getenv()` in config file, see https://github.com/ltb-project/self-service-password/issues/189

**Reply by evertonpavan** [reply]
Yes, I use getenv(), but show that erro: 'DAP - Bind error 49  (Invalid credentials)'

/ config.inc.local.php
...
$ldap_bindpw = getenv("PASSWORD_AD");

**Reply by coudot** [reply]
You need to load the env file, it is not done by default.
